src/main/java/lab3/Main.java

```diff
@@ -0,0 +1,33 @@
+package lab3;
+
+import lab3.game.Board;
+import lab3.ui.Console;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Board board = new Board();
+        Console console = new Console();
+        char currentPlayer = 'X';
+
+        while (true) {
+            console.displayBoard(board);
+            int[] move = console.getMove(board);
+
+            if (board.placeMove(move[0], move[1], currentPlayer)) {
```

Comment:
Using a record here like Position (int row, int col) or or even Position(Row row, Col col) with two enums Row and Col (see the sample solution) would make the code more intuitive as you wouldn't need to remember whether [0] or [1] is the row or col.